src/test/java/com/glovoapp/planningpoker/e2e/voting/AllVotesNumericalTest.java

```diff
@@ -0,0 +1,62 @@
+package com.glovoapp.planningpoker.e2e.voting;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class AllVotesNumericalTest {
+
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
+    MainPage firstUserMainPage = new MainPage(driver);
+    SessionPage firstUserSessionPage = new SessionPage(driver);
+    MainPage secondUserMainPage = new MainPage(driver2);
+    SessionPage secondUserSessionPage = new SessionPage(driver2);
+
+    private final String sessionName = "TEST29";
+    private final String userName1 = "michal";
```

Comment:
Constant should be `static`.

```suggestion
    private static final String FIRST_USERNAME = "michal";
```